MAINTAINERS.yml

```diff
@@ -1582,6 +1582,23 @@ nRF Platforms:
     labels:
         - "platform: nRF"
 
+Renesas R-Car Platforms:
+    status: maintained
+    maintainers:
+        - aaillet
+        - pmarzin
```

Comment:
Please have @pmarzin start as a collaborator first

Reply:
Wasn't sure about TSC roles assignment, just did it, thanks !